app/models/call_list_entry.rb

```diff
@@ -0,0 +1,20 @@
+# Object representing a patient's call list.
+class CallListEntry
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  # Relationships
+  belongs_to :user
+  belongs_to :patient
+
+  # Fields
+  field :order_key, type: Integer
+  field :line, type: String
+
+  # Validations
+  validates :order_key, :line, presence: true
+  validates :patient, uniqueness: { scope: :user }
+
+  # Indices
+  index line: 1
```

Comment:
(moving the file ate the comment on this)

the `1` seems to be the mongoid convention for whatever reason. See the docs: https://docs.mongodb.com/mongoid/current/tutorials/mongoid-indexes/